sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/implementation/OkHttpAsyncResponse.java

```diff
@@ -87,7 +87,7 @@ public Mono<InputStream> getBodyAsInputStream() {
             return Mono.empty();
         }
 
-        return Mono.using(responseBody::byteStream, Mono::just, ignored -> this.close(), false);
+        return Mono.just(responseBody.byteStream());
```

Comment:
Should we use `Mono.defer()` instead of `Mono.just()` here? We don't want to create the `InputStream` eagerly.